server/server.js

```diff
@@ -100,13 +100,6 @@ function createServer() {
     });
   });
 
-  server.post('/todos/changeCategory', (req, res) => {
-    res.setHeader('Access-Control-Allow-Origin', '*');
-    const {todoId, categoryId} = req.query;
-
-    db.changeCategory(todoId, categoryId).then(() => res.status(200).send('Category deleted successfully'));
```

Comment:
We should return 200, when we successfully update category. Change it in '/changeCategory' which you left.